src/esm.ts

```diff
@@ -176,7 +186,10 @@ export function createEsmHooks(tsNodeService: Service) {
       // Call the new defaultLoad() to get the source
       const { source: rawSource } = await defaultLoad(
         url,
-        { format },
+        {
+          format,
+          importAssertions: context.importAssertions,
+        },
```

Comment:
I'm aboard with that as long as I can be convinced that it's safe. I just played it over-safe initially since I didn't want to spend time considering it.

Reply:
In general, I also prefer playing it safe.  But I don't know what other fields might appear on the `context` object.  Do we know if node plans to add additional fields, and if so, what those fields are?

Reply:
How about we limit ourselves to forwarding only *known* properties, but we add a unit test that probes node.js and detects when it starts including additional properties on the context object?  Our test matrix runs nightly against many versions of node, including node nightly, so this will be a good way to receive advance warning when node starts including additional stuff on the context which we might want to forward?